config/routes.rb

```diff
@@ -72,7 +72,7 @@
   get '/partners/:id/events/:year/:month/:day' => 'partners#show', constraints: ymd
   get '/places' => 'partners#index' # Removing separate Places view for now.
   get '/partners/:id/embed' => 'places#embed'
-  post '/partners/tag' => 'partners#tag'
+  # post '/partners/tag' => 'partners#tag'
```

Comment:
this could be deleted

Reply:
If we're not using the turbo-stream thing any more lets also delete this template

https://github.com/geeksforsocialchange/PlaceCal/blob/46b3db85c5f4ed399ffd194581d3a13fa6af816f/app/views/partners/tag.turbo_stream.erb